integrations/elasticsearch/tests/test_bm25_retriever.py

```diff
@@ -38,6 +39,20 @@ def test_to_dict(_mock_elasticsearch_client):
             "document_store": {
                 "init_parameters": {
                     "hosts": "some fake host",
+                    "api_key": {
+                        "env_vars": [
+                            "ELASTIC_API_KEY",
+                        ],
+                        "strict": False,
+                        "type": "env_var",
+                    },
+                    "api_key_id": {
+                        "env_vars": [
+                            "ELASTIC_API_KEY_ID",
+                        ],
+                        "strict": False,
+                        "type": "env_var",
+                    },
```

Comment:
Could we also update the `from_dict` test below to add `api_key` and `api_key_id` in the dict that is being deserialized?